### DISCUSSION TITLE
What are your thoughts on wdd themes with more than 4 images?

### DISCUSSION THREAD

**Discussion by armaninyow** [t1m0thyj/WinDynamicDesktop#557] [Q&A]
I've made a few themes with 24 different images each. I used the specific sunrise and sunset times with 120 minute duration. I put 2 images in sunriseImageList and sunsetImageList so it means they appear an hour only. And to make it even i put 10 images in dayImageList and nightImageList so they also appear an hour. So basically, in every hour a different image will appear. Did i do it right?

**Comment by t1m0thyj** [comment]
Yes, this should work as you have described, and is a clever way to show a different image each hour. If all you want to do is update your wallpaper each hour, then you wouldn't need WinDD and could use the desktop slideshow feature built-in to Windows, but it sounds like you also want the schedule to be tied to the time of day.

A few of Apple's themes bundled with WinDD offer an example of defining a theme with more than 4 images, for example Mojave Desert:
```json
{
  "imageFilename": "Mojave_Desert_*.jpg",
  "imageCredits": "Apple",
  "sunriseImageList": [
    2,
    3,
    4
  ],
  "dayImageList": [
    5,
    6,
    7,
    8,
    9,
    10,
    11
  ],
  "sunsetImageList": [
    1,
    12,
    13
  ],
  "nightImageList": [
    14,
    15,
    16
  ]
}
```
